x-pack/plugins/cloud_integrations/cloud_experiments/common/constants.ts

```diff
@@ -21,6 +21,7 @@ export enum FEATURE_FLAG_NAMES {
    * It resolves the URL that the button "Add Integrations" will point to.
    */
   'security-solutions.add-integrations-url' = 'security-solutions.add-integrations-url',
+  'security-solutions.guided-onboarding-content' = 'security-solutions.guided-onboarding-content',
```

Comment:
Just 1 nit: can we add a comment to explain what this flag is for so we don't need to search through the code? 😇

Reply:
Done, thank you